components/script/dom/gpubuffer.rs

```diff
@@ -79,14 +88,21 @@ impl GPUBuffer {
         buffer: WebGPUBuffer,
         device: WebGPUDevice,
         state: GPUBufferState,
-        size: GPUSize64,
-        usage: u32,
+        size: u64,
```

Comment:
Nothing in particular. I had removed the size field initially while cleaning up the extra validation for `copyBuffertoBuffer`. Then, while implementing `mapAsync` I had to reintroduce it so I just used u64 instead of importing GPUSize64. I'll change it if it looks more appropriate.